### PR TITLE
Feat/add oif summit 2023 proposal scs five releases later

### PR DESCRIPTION
**Pull request by fkr**

### CODE REVIEW

File: OpenInfraSummit2023/SCS-Five-Releases-later.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Technical Level: Beginner
+
+## Abstract (1000 chars)
+
+The Sovereign Cloud Stack Project (SCS) started out in 2021 with the vision
+to create one platform — standardized, built and operated by many. The SCS project
+set out to create and nurture a set of standards and back this effort by providing
+a reference implementation based on OpenStack. 
+Since the project start there have been five releases, released twice a year in
```
Comment (itrich) [inline OpenInfraSummit2023/SCS-Five-Releases-later.md]: Since the project _started_

File: OpenInfraSummit2023/SCS-Five-Releases-later.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+Track: Public Cloud
+
+Authors: Felix Kronlage-Dammers
+
+Technical Level: Beginner
+
+## Abstract (1000 chars)
+
+The Sovereign Cloud Stack Project (SCS) started out in 2021 with the vision
+to create one platform — standardized, built and operated by many. The SCS project
```
Comment (itrich) [inline OpenInfraSummit2023/SCS-Five-Releases-later.md]: My sugestion: A growing community of more than 20 contributing companies set out to create and ...

File: OpenInfraSummit2023/SCS-Five-Releases-later.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+set out to create and nurture a set of standards and back this effort by providing
+a reference implementation based on OpenStack. 
+Since the project start there have been five releases, released twice a year in
+March and September. In the spirit of public money - public code the project utilizes
```
Comment (itrich) [inline OpenInfraSummit2023/SCS-Five-Releases-later.md]: PMPC and BMWK should be capitalized as they're titles/names.

File: OpenInfraSummit2023/SCS-Five-Releases-later.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+money from a funding by the german ministry for economics and climate action to coordinate
+and fund development in the various upstream projects.
+This talk will give an overview of what has been achieved over the course of the last years,
+will feature some insights into what has been easy and what was thougth to be easy but
```
Comment (itrich) [inline OpenInfraSummit2023/SCS-Five-Releases-later.md]: what was _thought_

File: OpenInfraSummit2023/SCS-Five-Releases-later.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+turned out to be hard.
+It will highlight how the project works upstream and how the corresponding results
+enable Cloud Service Providers to deliver a better product to their customers and users.
+
+## Social Summary (100 chars)
+
+Sovereign Cloud Stack not only defined standards - but enables CSPs. Let's show how!
```
Comment (itrich) [inline OpenInfraSummit2023/SCS-Five-Releases-later.md]: not only *defines* standards

File: OpenInfraSummit2023/SCS-Five-Releases-later.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+## What should attendees expect to learn? (1000 chars)
+
+Aside from sheding some light into the progress of the efforts of the Sovereign
```
Comment (itrich) [inline OpenInfraSummit2023/SCS-Five-Releases-later.md]: into the progress ~~of the efforts~~ of the Sovereign Cloud Stack

**Review by itrich — CHANGES_REQUESTED** [review]

File: OpenInfraSummit2023/SCS-Five-Releases-later.md (hunk 2 above)
Comment (fkr) [reply OpenInfraSummit2023/SCS-Five-Releases-later.md]: Can you take a look if what I made out of now, matches your idea?

**Comment by fkr** [comment]
thanks @itrich for the initial review, please re-check ;)

**Review by itrich — APPROVED** [review]
LGTM! Thank you for incorporating my feedback. The proposal reads very well and interesting!

**Pull request merged**